docs-ref-conceptual/includes/cli-install-linux-apt.md

```diff
@@ -52,16 +52,18 @@ If you prefer a step-by-step installation process, complete the following steps
 2. Download and install the Microsoft signing key:
 
     ```bash
+    sudo mkdir -p /etc/apt/keyrings
     curl -sL https://packages.microsoft.com/keys/microsoft.asc |
         gpg --dearmor |
-        sudo tee /etc/apt/trusted.gpg.d/microsoft.gpg > /dev/null
+        sudo tee /etc/apt/keyrings/microsoft.gpg > /dev/null
+    sudo chmod go+r /etc/apt/keyrings/microsoft.gpg
```

Comment:
`sudo chmod go+r /etc/apt/keyrings/microsoft.gpg` seems to be a no-op.

``` bash
curl -sL https://packages.microsoft.com/keys/microsoft.asc | sudo gpg --dearmor -o ./kk.gpg
ll kk.gpg
-rw-r--r-- 1 root root 641 Apr 11 10:20 kk.gpg
```

Reply:
Wouldn't be with a modified `umask`. Explicitly setting it to `go+r` will however guarantee it's readable.